docs/ir/ir-breaking-changes.rst

```diff
@@ -0,0 +1,7 @@
+********************************
+Solidity ir Breaking Changes
+********************************
+
+This section highlights the main breaking changes introduced in Solidity after
```

Comment:
There will probably be a transition phase where you can enable it, so maybe better to just write "difference between the old and the IR-based codegen" or something like that.

Reply:
I changed title and the text as well